sktm/jenkins.py

```diff
@@ -23,7 +23,8 @@
 
 class JenkinsProject(object):
     """Jenkins project interface"""
-    def __init__(self, name, url, username=None, password=None):
+    def __init__(self, name, url, username=None, password=None,
+                 retry_cnt="30"):
```

Comment:
I don't think this will work (that said, the `username` and `password` params don't need the defaults either) -- if the option is not present, `cfg.get('jretry')` will return `None`, which is a perfectly valid value, and since the option will be passed the default is never used.

Instead, you should either assign a default value at the end of the `load_config` method (if the option is not `None`) or in this method (the first one is preferred)

Reply:
Does this change work fine? (I didn't update load_config() yet)
-    parser.add_argument("--jretry", help="Counter to retry Jenkins")
+    parser.add_argument("--jretry", help="Counter to retry Jenkins",
+                        type=int, default=30)

Reply:
This would work for command line options but not the config file. You'll still need to add the logic for retyping it because of that

Reply:
OK, Added.